teos/src/config.rs

```diff
@@ -242,6 +242,94 @@ impl Config {
     pub fn is_default(&self) -> bool {
         self == &Config::default()
     }
+
+    /// checks non-default options using Vector
```

Comment:
I'd replace this by something along the lines of:

Gets a collection of custom options